test/sanity/validate-modules/ignore.txt

```diff
@@ -324,6 +324,7 @@ lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py E322
 lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py E323
 lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py E324
 lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py E326
+lib/ansible/modules/cloud/spotinst/spotinst_mrScaler.py E326
```

Comment:
New ignores should not be added. Correct the underlying issue in the module. If the issue is in an existing module_utils, fix that issue in a separate PR first.